src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -173,8 +172,21 @@ internal partial class CodeFixService : ICodeFixService
             // invariant: later code gathers & runs CodeFixProviders for diagnostics with one identical diagnostics span (that gets set later as CodeFixCollection's TextSpan)
             // order diagnostics by span.
             SortedDictionary<TextSpan, List<DiagnosticData>>? aggregatedDiagnostics = null;
+
+            // For 'CodeActionPriorityRequest.Normal' or 'CodeActionPriorityRequest.Low', we do not compute suppression/configuration fixes,
+            // those fixes have a dedicated request priority 'CodeActionPriorityRequest.Lowest'.
+            // Hence, for Normal or Low priority, we only need to execute analyzers which can report at least one fixable diagnostic
+            // that can have a non-suppression/configuration fix.
+            var shouldIncludeDiagnostic = priority is CodeActionRequestPriority.Normal or CodeActionRequestPriority.Low
+                ? GetFixableDiagnosticFilter(document)
+                : null;
```

Comment:
hrmm...what happens if we have CodeActionRequestPriority.High though?  should this just be `if priority is not CodeActionRequestPriority.Lowest`? 

If this logic is correct can you expand on the comment to explain the handling of 'High'?